BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-134: Create Spinner shared component

- create spinner shared component
- replace old spinner with new one
- implement it in all the old code

How to test?
- use the app as usual and inspect it with the redux dev tool
- all the instances where the spinner (isLoading) is needed the new spinner should appear

note: the position of the spinner will vary on the different screens. This will be fixed when all the other aesthetic changes are made
